README.md

```diff
@@ -274,8 +274,9 @@ Will produce this `valohai.yaml` config:
 
 If you wish to further develop `valohai-utils`, remember to install development dependencies and write tests for your additions.
 
+Linting is done via `pre-commit`, so be sure to set that up.
```

Comment:
we could be more helpful by specifying how to set it up :P 

e.g.

```
pip install pre-commit
pre-commit install
```

also, could mention that lints can be manually ran with something like `pre-commit run --all-files` (or `make lint` if you add that target to Makefile)